File: cpu/computation.go
package main

import (
	"fmt"
	time "time"
)

func main() {
	size := 100000000
	data := make([]int, size)
	for i := 0; i < size; i++ { data[i] = i }

	start := time.Now()
	sum := 0
	for _, v := range data {
		sum += v
	}

	fmt.Printf("Go: Sum %d in %v\n", sum, time.Since(start))
}

File: cpu/computation.rs
use std::time::Instant;

fn main() {
    let size = 100000000;
    let mut data = vec![0; size];
    for i in 0..size { data[i] = i; }

    let start = Instant::now();
    let sum: usize = data.iter().sum();

    println!("Rust: Sum {:?} in {:?}", sum, start.elapsed());
}
